src/js/covidMap.js

```diff
@@ -583,27 +788,67 @@ fluid.defaults("fluid.styledButton", {
     }
 });
 
+fluid.defaults("fluid.backButton", {
+    gradeNames: "fluid.button",
+    modelListeners: {
+        "resetQuery": {
+            path: "activate",
+            changePath: "{map}.model.query",
+            value: ""
+        },
+        "resetSelectedIndex": {
+            path: "activate",
+            changePath: "{map}.model.selectedIndex",
+            value: null
+        }
+    }
+});
+
+// The cities list component
 fluid.defaults("fluid.covidMap.citiesList", {
     gradeNames: ["fluid.viewComponent"],
     markup: {
         cityTemplate: "<div class=\"fl-mapviz-city fl-mapviz-hoverable-focusable\">%city</div>"
     },
     selectors: {
+        title: ".fl-mapviz-city-list-title",
         cities: ".fl-mapviz-cities",
-        element: ".fl-mapviz-city"
+        element: ".fl-mapviz-city",
+        expandButton: ".fl-mapviz-expand-collapse-button"
     },
     modelListeners: {
         render: {
             path: "cities",
             func: "{that}.renderMarkup"
         }
     },
+    components: {
+        expandButton: {
+            type: "fluid.expandButton",
+            container: "{citiesList}.dom.expandButton",
+            options: {
+                model: {
+                    expanded: "{map}.model.expandedPanels.citiesList"
+                },
+                modelRelay: {
+                    collapseTitle: {
```

Comment:
In the other expandButton these rules were prefixed "visible" rather than "collapse"